astropy/utils/decorators.py

```diff
@@ -289,8 +290,9 @@ def deprecated_renamed_argument(old_name, new_name, since,
     old_name : str or list/tuple thereof
         The old name of the argument.
 
-    new_name : str or list/tuple thereof
-        The new name of the argument.
+    new_name : str or list/tuple thereof or `None`
+        The new name of the argument. ``old_name`` has been deprecated to
+        be removed if `None`.
```

Comment:
Maybe rephrase to ``The new name of the argument. Set this to `None` if removing the argument instead of renaming.``